python/tests/conftest.py

```diff
@@ -686,7 +700,9 @@ def spawn_azurite(azurite_port):
 @pytest.fixture(
     scope="function",
     params=(
-        ["moto_s3_uri_incl_bucket", "azurite_azure_uri_incl_bucket"] if AZURE_SUPPORT else ["moto_s3_uri_incl_bucket"]
+        ["moto_s3_uri_incl_bucket", "azurite_azure_uri_incl_bucket", "mongo_test_uri"]
+        if AZURE_SUPPORT
+        else ["moto_s3_uri_incl_bucket", "mongo_test_uri"]
```

Comment:
We don't have a simple way to start a Mongo server in all environments, so I wouldn't add it to test parameters.

It's tested internally, and we are not fully supporting it for external use at the moment.

Reply:
We need to add it to the tests. The whole point of this change is to start supporting it for external use.